pruning/manager.go

```diff
@@ -135,7 +135,7 @@ func (m *Manager) HandleHeight(previousHeight int64) int64 {
 
 	if int64(m.opts.KeepRecent) < previousHeight {
 		pruneHeight := previousHeight - int64(m.opts.KeepRecent)
-		// We consider this height to be pruned iff:
```

Comment:
This is intentional I think

(`iff` is common short hand for if and only if in math / logical proof contexts)

Reply:
correct :P